File: Upcoder/Lập trình hướng đối tượng/1.Object and Class/Lớp phân số cơ bản/LOP1.1.PhanSo.cpp
#include<bits/stdc++.h>
using namespace std;
class PhanSo {
private:
    int tuSo;   // Tử số
    int mauSo;  // Mẫu số
public:
    void nhap();
    void xuat();
};

// Hàm chính
int main() {
    PhanSo ps;
    ps.nhap();
    ps.xuat();
    return 0;
}

void PhanSo::nhap()
{
    cin >> tuSo >> mauSo;
}

void PhanSo::xuat()
{
    cout << tuSo << "/" << mauSo;
}

File: Upcoder/Lập trình hướng đối tượng/1.Object and Class/Lớp phân số cơ bản/LOP1.1.PhanSo.cs
using System;
namespace PSo
{
    public class PhanSo
    {
        private int tuSo;
        private int mauSo;

        public void Nhap()
        {
            string[] input = Console.ReadLine().Split();
            tuSo = int.Parse(input[0]);
            mauSo = int.Parse(input[1]);
        }

        public void Xuat()
        {
            Console.WriteLine($"{tuSo}/{mauSo}");
        }
    }
    public class Program
    {
        static void Main(string[] args)
        {
            PhanSo ps = new PhanSo();
            ps.Nhap();
            ps.Xuat();
        }
    }
}